src/main/resources/application-prod.yml

```
@@ -21,3 +21,7 @@ spring:
         default_batch_fetch_size: 1000
         dialect: org.hibernate.dialect.MySQLDialect
     open-in-view: false
+  data:
+    redis:
+      host: redis # redis container 이름
+      port: 6379
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)